packages/cozy-procedures/src/redux/documentsDataSlice.js

```diff
@@ -91,7 +91,7 @@ const getCompletedCount = state =>
   Object.values(getData(state)).reduce((acc, { files }) => {
     files.map(file => {
       if (file !== undefined) {
-        acc++
+        acc++ // eslint-disable-line no-param-reassign
```

Comment:
in this case, I guess it we should not disable the line but fix the reassignment.

Reply:
I get something let this should work
```
const getCompletedCount = state => {
  let count = 0
  Object.values(getData(state)).forEach({ files }) => {
    files.map(file => {
      if (file !== undefined) {
        count++ 
        }
    })
})
return count
}
```

Reply:
As I don't know this lib I preferred to disable each error rather than fix it (fear of unintended side-effects)

Reply:
Without needing to rewrite the entire method, we can keep the reduce and put the variable in the reduce:
```js
Object.values(getData(state)).reduce((acc, { files }) => {
    let newAcc = acc
    files.map(file => {
      if (file !== undefined) {
        newAcc++ // eslint-disable-line no-param-reassign
      }
    })
    return newAcc
  }, 0)
```

If we want to simplify the method, here is another try:
```js
Object.values(getData(state))
  .flatMap(data => data.files)
  .filter(file => filer !== undefined)
  .length
```

But I agree with @acezard, the PR's goal is not to fix the errors yet. We should try to fix them only if we have enough time to test the resulting behavior on project that use this method.

Now the real question is : why do we have this change related to `no-param-reassign` if the PR is named `add default-param-last`? Should we rename the PR/commit?

Reply:
@Ldoppea I admit I had no idea why this error popped, I didn't touch it and eslint didn't change version.